go.mod

```diff
@@ -2,9 +2,18 @@ module github.com/filecoin-project/go-f3
 
 go 1.20
 
+require github.com/stretchr/testify v1.8.4
+
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/ipfs/go-cid v0.0.5 // indirect
+	github.com/minio/blake2b-simd v0.0.0-20160723061019-3f5f724cb5b1 // indirect
+	github.com/minio/sha256-simd v0.1.1-0.20190913151208-6de447530771 // indirect
+	github.com/mr-tron/base58 v1.1.3 // indirect
+	github.com/multiformats/go-base32 v0.0.3 // indirect
+	github.com/multiformats/go-multibase v0.0.1 // indirect
+	github.com/multiformats/go-multihash v0.0.13 // indirect
+	github.com/multiformats/go-varint v0.0.5 // indirect
```

Comment:
It looks like there's some noise crept in here, please normalise before merging.

Reply:
Done.